app/client/src/components/StartButton.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div class="start-analysis">
+    <button v-if='!analysisStatus.submitted && allSketched' class="btn btn-block btn-standard"
+    @click='runPoppunk();setStatus({ task: "submitted", data: "submitted"})'>
```

Comment:
If you need a multi-line handler, it's probably more readable to put it in a method e.g. 'handleSubmit' and call that.